.github/workflows/pull_request.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
To really change the versions that are supported, you need to change the `setup.py` file 😉

Reply:
I see that python3.9 is declared [here](https://github.com/pdyba/lambdalizator/blob/fa1d26e88e4911f66f72dd5c21550587ec1936dd/examples/rest/simple_resource.py#L1) too.